shawmut/weather.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from requests_forecast import Forecast
+from shawmut.settings import conf
+
+class ShawmutWeather(object):
+    MAX_RETRIES = 500 # We are allowed 1000 API calls per today so this is pretty conservative
+
+    def __init__(self):
+        self.refresh_todays_data()
+
+    def get_todays_data(self):
+        if datetime.now() > self.todays_datetime():
```

Comment:
This only works if `todays_data['time']` is the very end of the day, right? Otherwise we might end up refreshing any time afternoon. I think you can make it a date object instead of a datetime object and that will do what you want.